qttasserver/contrib/qttas-server.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=QTTAS Service
+
+[Service]
+ExecStart=/usr/bin/qttasserver
```

Comment:
This is where qttasserver gets installed from debian package but dunno if there should be any parameters passed ?

Reply:
this is ok